site/app/libraries/database/DatabaseQueriesPostgresql.php

```diff
@@ -677,6 +677,39 @@ public function getTotalUserCountByGradingSections($sections, $section_key) {
         }
         return $return;
     }
+    
+    public function getTotalSubmittedUserCountByGradingSections($g_id, $sections, $section_key) {
+        $return = array();
+        $params = array();
+        $where = "";
+        if (count($sections) > 0) {
+            // Expand out where clause
+            $sections_keys = array_keys($sections);
+            $where = "WHERE {$section_key} IN (";
+            foreach($sections_keys as $section)
+                $where .= ($section+1) . ($section != $sections_keys[count($sections_keys)-1] ? "," : "");
+            $where .= ")";
```

Comment:
You should be putting "?" for parameters in your query that get filled in by PDO. This helps for escaping variables and ensuring no potential accidental SQL injections happen. The way this would work is that for every variable in an array, you put a "?" in the query where you want it to be inserted, pass the array to query() and it happens for you (you can see a similar usage of this pattern elsewhere in this file where I used array_fill and implode to get the right number of ? for a IN where clause.